### DISCUSSION TITLE
[功能请求]:添加下课准备提醒

### DISCUSSION THREAD

**Discussion by xinya28** [ClassIsland/ClassIsland#558] [插件功能请求]
### 检查清单

- [X] 我已经在 [Issues](https://github.com/HelloWRC/ClassIsland/issues?q=) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已经详细阅读过应用的帮助文档，但我想要的功能没有实现。
- [X] 我已确认我的功能尚未在 [dev](https://github.com/HelloWRC/ClassIsland/commits/dev/) 分支实现。

### 背景与动机

提醒教学进度
(渴望下课，看会进度条续命
(不想让老师拖堂(bushi

### 想要实现或优化的功能

像上课前发出准备提醒一样，提醒下课[doge]
![image](https://github.com/HelloWRC/ClassIsland/assets/140145748/b7df7e82-0576-43b9-b15f-8b2d7619ebb2)

**Comment by jiangyin-srinternet** [comment]
可以使用下个课间的课前提醒功能，该提醒会侵入上一节课。

**Comment by xinya28** [comment]
非常感谢,不过似乎不起作用(无法侵入上一节课),这或许是由于我的设置不正确.
请求具体设置方式

**Comment by HelloWRC** [comment]
感觉这个功能可能有点侵入课堂了，容易影响上课。这个功能目前不会加入到应用本体，可能会考虑以插件的形式提供。

**Comment by FrankXiDong** [comment]
现在或许自动化可以实现了？
上课触发，然后wait37分钟，行动`提醒`
